inst/BS5/templates/content-reference-index.html

```diff
@@ -5,7 +5,7 @@
       <h1>{{{pagetitle}}}</h1>
     </div>
 
-    {{#rows}}<div id="{{slug}}" class="section level2">
+    {{#rows}}<div {{#slug}}id="{{.}}"{{/slug}} class="section level2">
```

Comment:
what's the difference?

Reply:
If there's no slug, we now don't add an `id`.